docs/builders/overview.mdx

```diff
@@ -0,0 +1,91 @@
+---
+description: >
+  The Nutanix packer plugin overview
+page_title: Nutanix plugin Overview
+nav_title: Overview
+---
+# Nutanix Plugin Overview
+
+<!--
+  Include a short overview about the plugin.
+
+  This document is a great location for creating a table of contents for each
+  of the components the plugin may provide. This document should load automatically
+  when navigating to the docs directory for a plugin.
+
+-->
+The `Nutanix` multi-component plugin can be used with HashiCorp [Packer](https://www.packer.io)
+to create custom images.
+
+## Installation
+
+### Using pre-built releases
+
+#### Using the `packer init` command
+
+Starting from version 1.7, Packer supports a new `packer init` command allowing
+automatic installation of Packer plugins. Read the
+[Packer documentation](https://www.packer.io/docs/commands/init) for more information.
+
+To install this plugin, copy and paste this code into your Packer configuration .
+Then, run [`packer init`](https://www.packer.io/docs/commands/init).
+
+```hcl
+packer {
+  required_plugins {
+    nutanix = {
+      version = ">= 0.2.0"
```

Comment:
0.3.0 will be the first release which supports all of the documented features

Reply:
Fixed